kpi/utils/mongo_helper.py

```diff
@@ -18,8 +18,23 @@ class MongoHelper:
     and KoBoCAT's ParseInstance class to query mongo.
     """
 
-    KEY_WHITELIST = ['$or', '$and', '$exists', '$in', '$gt', '$gte',
-                     '$lt', '$lte', '$regex', '$options', '$all']
+    OR_OPERATOR = '$or'
+    AND_OPERATOR = '$and'
+    IN_OPERATOR = '$in'
```

Comment:
just curious why these operators get their own constants and others do not

Reply:
Oli: it improved readability in some places where code was written with many operators